src/main/java/org/apache/commons/collections4/collection/CompositeCollection.java

```diff
@@ -313,6 +315,27 @@ public boolean removeAll(final Collection<?> coll) {
         return changed;
     }
 
+    /**
+     * Removes all of the elements of this collection that satisfy the given predicate from this composite collection.
+     * <p>
+     * This implementation calls <code>removeIf</code> on each collection.
+     *
```

Comment:
You must close tags.

Reply:
Are you referring to the p tag?
I thought it was a convention because there are p tags in comments of other methods.
Do you want to commit and delete other p tags?

Reply:
Yes, starting in Java 8, Javadoc does not accept by default malformed XHTML, so you must close <p> tags with a </p> to create a paragraph. Just because the current Javadoc still has not been cleaned up does not mean we want to create _more_ defective comments ;-)